src/store/Store.js

```diff
@@ -1,5 +1,7 @@
 import assignIn from 'lodash/assignIn';
 
+import { default as applyMiddlewareFn } from './applyMiddleware';
```

Comment:
Quick question here: Why import it twice (once into store, and then again into index file) vs just importing the `applyMiddleware` straight into the index file? Am I missing something?

Reply:
^ good question

Reply:
When a colleague and I were working on this we had some issue that lead us to exposing it through Store, but I honestly can't quite recall what that was now. Let me rack my brain/touch base with him today and I'll update this comment.

Reply:
@davidtom sounds good! Not a huge issue, but I figured we should check if we can clean it up. Looking forward to hearing more.

Reply:
Yeah, I think cleaning it up would be the best option. After reflecting on it a bit we did the imports this way because we were having issues importing it/making it accessible via the index file. If you're able to get that working this line can definitely be removed.

Reply:
@davidtom Sounds good! I'll include that in my PR.